### PR TITLE
Parameter checks

### PR DESCRIPTION
**Pull request by schmeat**
We're finding blues clues!

### CODE REVIEW

File: vickibot.py [hunk 1]
```diff
@@ -38,15 +35,47 @@ def __init__(self):
     #        json.dump(self.stats, jsonFile)
 
     def buyOrderKrakenETHBTC(self,volume,price):
+        if isinstance(volume, int) == False:
```
Comment (spood) [inline vickibot.py]: should be isinstance(volume, Decimal) for volume and price

File: vickibot.py [hunk 2]
```diff
@@ -38,15 +35,47 @@ def __init__(self):
...
         print("Actually placing buy limit order for ETH with BTC at price",price,"with volume",volume)
+
         self.kraken.query_private('AddOrder', {'pair': 'XETHXXBT',
                                  'type': 'buy',
                                  'ordertype': 'limit',
                                  'price': price,
                                  'volume': volume})
 
     def sellOrderKrakenETHBTC(self,volume,price):
+        if isinstance(volume, int) == False:
```
Comment (spood) [inline vickibot.py]: same

Comment (schmeat) [reply vickibot.py]: Done

File: vickibot.py (hunk 1 above)
Comment (schmeat) [reply vickibot.py]: Done

**Pull request merged**